data/PRIME/data.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Prime",
+  "symbol": "PRIME",
+  "decimals": 18,
+  "description": "PRIME is a utility token for games in the Echelon ecosystem such as Parallel TCG, Parallel Colony, and more.",
```

Comment:
Could you add that this token is bridged with layer-0 here?  otherwise looks good.

Reply:
LGTM once this is done